### PR TITLE
updating failing specs

### PR DESCRIPTION
**Pull request by banukutlu**

### CODE REVIEW

File: spec/features/availability_spec.rb [hunk 1]
```diff
@@ -82,6 +82,7 @@
   describe 'Archival Material:' do
     it 'has a \'Request Material\' link so it can be requested through Aeon', js: true do
       visit '/catalog/1836205'
+      click_button('View More')
```
Comment (cdmo) [inline spec/features/availability_spec.rb]: Is this necessary? Box 1 the first one in the first 4 listed...

File: spec/features/availability_spec.rb [hunk 2]
```diff
@@ -65,13 +65,13 @@
   describe 'User opens a record with many holdings in a library' do
     it 'toggles 4 or more holdings with View More/View Less button', js: true do
       visit '/catalog/1793712'
-      within 'div[data-library="UP-MICRO"]' do
+      within 'div[data-library="UP-PAT"]' do
```
Comment (cdmo) [inline spec/features/availability_spec.rb]: Curious. Did we get rid of our micro collection then?

Comment (banukutlu) [reply spec/features/availability_spec.rb]: not sure if we got rid of that collection but this record does not have UP-MICRO anymore

File: spec/features/availability_spec.rb (hunk 1 above)
Comment (banukutlu) [reply spec/features/availability_spec.rb]: The reason I added that click is to make sure that the holding we are testing here would always be in the visible css. With being not 💯 sure, I think this is the spec that is failing time to time depending on the order of holdings listed, so I thought adding a click to view more would work around that.

Comment (cdmo) [reply spec/features/availability_spec.rb]: Ok, cool. Thanks.

**Pull request merged**